plugins/cros-ec/fu-cros-ec-firmware.c

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright (C) 2020 Benson Leung <bleung@chromium.org>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#include "config.h"
+
+#include "fu-fmap-firmware.h"
+#include "fu-cros-ec-common.h"
+#include "fu-cros-ec-firmware.h"
+
+#define MAXSECTIONS	2
+
+struct _FuCrosEcFirmware {
+	FuFirmware			parent_instance;
```

Comment:
I think this needs to be `FuFmapFirmware parent_instance`, no?

Reply:
Done.